exercises/simple-linked-list/linked_list_test.exs

```diff
@@ -0,0 +1,155 @@
+if !System.get_env("EXERCISM_TEST_EXAMPLES") do
+  Code.load_file("linked_list.exs", __DIR__)
+end
+
+ExUnit.start
+ExUnit.configure exclude: :pending, trace: true
+
+defmodule LinkedListTest do
+  use ExUnit.Case
+
+  test "length/1 of new list" do
+    list = LinkedList.new()
+    assert LinkedList.length(list) == 0
+  end
+
+  @tag :pending
+  test "empty?/1 of new list" do
+    list = LinkedList.new()
+    assert LinkedList.empty?(list)
+  end
+
+  @tag :pending
+  test "length/1 of list of 1 datum" do
+    list = LinkedList.new() |> LinkedList.push(10)
+    assert LinkedList.length(list) == 1
+  end
+
+  @tag :pending
+  test "empty?/1 of list of 1 datum" do
+    list = LinkedList.new() |> LinkedList.push(20)
+    refute LinkedList.empty?(list)
+  end
+
+  @tag :pending
+  test "peek/1 of list of 1 datum" do
+    list = LinkedList.new() |> LinkedList.push(20)
+    assert LinkedList.peek(list) == {:ok, 20}
+    assert LinkedList.length(list) == 0
```

Comment:
Ok, you'll have to bear with me since my data structures may be a little weak, but isn't `peek` typically non-destructive in a linked list? So in this case, after peeking at the first element in the list, the length of that list should remain unchanged - in this case, 1?

I also might be totally wrong, so forgive me if that's the case!